pkg/actions/controller.go

```diff
@@ -0,0 +1,25 @@
+package actions
+
+import (
+	"context"
+
+	"github.com/solo-io/squash/pkg/api/v1"
+	"github.com/solo-io/squash/pkg/utils"
+)
+
+type UserController struct {
+	ctx      context.Context
+	daClient *v1.DebugAttachmentClient
+}
+
+func NewUserController() (UserController, error) {
```

Comment:
i see this is used just in tests..  should this be in a test package?

Reply:
what is the prupsoe of this struct?

Reply:
the implementation is a bit of a WIP - the idea is that that struct has (or will have) all the methods that the user is able to call from the command line